resources/carrier/en/234.txt

```diff
@@ -139,6 +142,7 @@
 2346987|Starcomms
 2346988|Starcomms
 2346989|Starcomms
+234700|Alpha Technologies
```

Comment:
We do accept only the mobile ranges in carrier file. 700 and 800 ranges belong to Alpha Technologies. As per information in http://www.alphatechnologieslimited.com/, this is not a mobile carrier.